src/Tizen.Theme.Common/Handler/CommonUIExtensions.cs

```diff
@@ -47,7 +44,8 @@ public static MauiAppBuilder UseTizenThemeCommonUI(this MauiAppBuilder builder)
                     //handlers.AddCompatibilityRenderer(typeof(OverlayMediaView), typeof(OverlayViewRenderer));
 
                     //TODO
-                    //handlers.AddCompatibilityRenderer(typeof(AnimatedNavigationPage), typeof(AnimatedNavigationPageRenderer));
+                    //handlers.AddHandler<AnimatedNavigationPage, AnimatedNavigationPageHandler>();
+                    handlers.AddHandler<AnimatedNavigationPage, AnimatedNavigationPageHandler>();
```

Comment:
Is this line also a change relate to ContentPopup? Is this an intentional or not?

Reply:
The code for other control I missed are included in this patch.
Thank you, I've fixed :)